packages/components/src/components/tag/tag.tsx

```diff
@@ -8,14 +8,22 @@ import classNames from "classnames";
   shadow: true
 })
 export class Tag {
-  /** (optional) Button variant */
+  /** (optional) Tag variant */
   @Prop() variant?: string = "";
+  /** (optional) Tag on an <a> element */
+  @Prop() link?: boolean = false;
 
```

Comment:
pill be an own prop - a boolean, and produce a dedicated state class: 'tag--pill'; 
variant prop is for variations like: 'primary', 'secondary', 'danger' etc.